code/modules/mining/equipment/mining_tools.dm

```diff
@@ -37,13 +37,40 @@
 	desc = "A silver-plated pickaxe that mines slightly faster than standard-issue."
 	force = 17
 
+/obj/item/weapon/pickaxe/gold
+	name = "gold-tipped pickaxe"
+	icon_state = "gpickaxe"
+	item_state = "gpickaxe"
+	digspeed = 17
+	origin_tech = "materials=4;engineering=4"
+	desc = "A pickaxe with a golden pick head."
+	force = 16
+
+/obj/item/weapon/pickaxe/uranium
+	name = "uranium-tipped pickaxe"
+	icon_state = "upickaxe"
+	item_state = "upickaxe"
+	digspeed = 30
+	origin_tech = "materials=4;engineering=4;"
+	desc = "A pickaxe with a uranium pick head. It's a bit slower, but it hits harder!"
+	force = 20
+
+/obj/item/weapon/pickaxe/plasma
+	name = "plasma-tipped pickaxe"
+	icon_state = "ppickaxe"
+	item_state = "ppickaxe"
+	digspeed = 14
+	origin_tech = "materials=2;engineering=3;plasmatech=2"
+	desc = "A pickaxe with a solidified plasma pick head. Extremely robust at cracking rock walls and digging up dirt."
+	force = 17
+
 /obj/item/weapon/pickaxe/diamond
 	name = "diamond-tipped pickaxe"
 	icon_state = "dpickaxe"
 	item_state = "dpickaxe"
-	digspeed = 14
+	digspeed = 11
```

Comment:
Plasma pick has the stats of the current diamond pick, and the diamond pick was made better (faster), giving it more value as an upgrade / item drop.